src/components/quiz-question/multi-choice/multi-choice.scss

```diff
@@ -9,6 +9,7 @@
   .questionText {
     @include ivq-text;
     flex-shrink: 0;
+    line-height: 21px;
```

Comment:
maybe wrap it with a selector so we don't repeat it 3 times?

Reply:
it related to "font-weight: bold".
we'll repeat selector same times as a "line-height", I don't think we'll get any benefit from it